tests/cypress/integration/delete-all-terms.test.js

```diff
@@ -0,0 +1,72 @@
+describe('Command: deleteAllTerms', () => {
+  beforeEach(() => {
+    cy.login();
+  });
+
+  after(() => {
+    // Restore default 20 items per page
+    cy.visit(`/wp-admin/edit-tags.php?taxonomy=category`);
+    cy.get('#show-settings-link').click();
+    cy.get('input.screen-per-page').click().clear().type(20);
+    cy.get('#screen-options-apply').click();
+
+    cy.visit(`/wp-admin/edit-tags.php?taxonomy=post_tag`);
+    cy.get('#show-settings-link').click();
+    cy.get('input.screen-per-page').click().clear().type(20);
+    cy.get('#screen-options-apply').click();
+  });
+
+  it('Should be able to Delete All Categories', () => {
+    cy.createTerm('Term to delete');
+    cy.createTerm('Term to delete 2');
+    cy.deleteAllTerms();
+    cy.get('.notice').should('contain', 'Categories deleted');
+    cy.get('#the-list a.row-title').should('have.length', 1);
+    cy.get('#the-list a.row-title')
+      .first()
+      .should('have.text', 'Uncategorized');
+  });
+
+  it('Should be able to delete paginated categories', () => {
+    for (let index = 0; index < 10; index++) {
+      cy.createTerm('Multi page category ' + index);
+    }
+    // Set 2 items per page
+    cy.get('#show-settings-link').click();
+    cy.get('input.screen-per-page').click().clear().type(2);
+    cy.get('#screen-options-apply').click();
+
+    cy.deleteAllTerms();
+    cy.get('#the-list a.row-title').should('have.length', 1);
+    cy.get('#the-list a.row-title')
+      .first()
+      .should('have.text', 'Uncategorized');
+  });
+
+  it('Should be able to Delete All Tags', () => {
+    cy.createTerm('Tag to delete', 'post_tag');
+    cy.createTerm('Tag to delete 2', 'post_tag');
+    cy.deleteAllTerms('post_tag');
+    cy.get('.notice').should('contain', 'Tags deleted');
+    cy.get('#the-list a.row-title').should('have.length', 0);
+  });
+
+  it('Should be able to delete paginated tags', () => {
+    for (let index = 0; index < 10; index++) {
+      cy.createTerm('Multi page tag ' + index, 'post_tag');
+    }
```

Comment:
What do you think about running WP CLI command here to create multiple terms? We can use [cy.exec](https://docs.cypress.io/api/commands/exec#Syntax) to run npm commands.

Reply:
I've noticed existing `wpCli` commands in ElasticPress which is scheduled for importing, let's get back to this and refactor the tests after it will be implemented.

I also have concern about the speed. From my past experience running shell commands in Cypress takes significantly longer then working through browser interface because of multi-level wrap: `wp-cli` wrapped by `wp-env` docker container then wrapped by `npm run` finally wrapped by cypress process ``cy.exec(`npm run env run tests-cli "wp term list category"`)``

Reply:
From other hand, I see that paginated tests take a lot of time. Probably adding xml fixtures and importing them with wp-cli will be faster anyway.

Reply:
@cadic Make sense to me. Feel free to merge this PR then 😄